src/App.tsx

```diff
@@ -26,70 +27,133 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((g1, g2) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return g1.localeCompare(g2);
+
+      case SortType.LENGTH:
+        return g1.length - g2.length;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
Don't start sorting if you don't have sort type + don't check sort type inside iterator, check it before

```suggestion
switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```